docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -367,17 +236,15 @@ az devops invoke --http-method PUT --area build --resource definitions --route-p
 
     * Open the project from your favorite IDE and navigate to infrastructure templates `./infra/templates` directory.
     * Manually delete template directories not needed for your enterprise.
+    * * The CI/CD pipeline needs to detect a code change to run the template-specific build and release jobs (in their respective stages). To force the template build and release to run, you may add a `FORCE_RUN` environment variable with a value of `true` to your *devint Environment Variables* variable group. You may also add a comment or extra line to a TF or Go file within the template in order for the pipeline script to detect a change without adding any additional override flags.
+
```

Comment:
Same comment as above, would a screenshot here help? many of these things seem option such as adding an extra line to tf or go file to detect a chance, etc so perhaps that might not make the most sense for screenshot but just a thought.

Reply:
I'm excluding the screenshot.